src/Mapper/CustomFieldsModulesMapper.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Canvas\Mapper;
+
+use AutoMapperPlus\CustomMapper\CustomMapper;
+use function Canvas\Core\isJson;
+use Phalcon\Mvc\Model\Resultset;
+
+class CustomFieldsModulesMapper extends CustomMapper
+{
+    /**
+     * @param Canvas\Models\FileSystem $file
+     * @param Canvas\Dto\Files $fileDto
+     * @return Files
+     */
+    public function mapToObject($customFieldsModules, $customFieldsModulesDto, array $context = [])
+    {
+        $customFieldsArray = [];
+        $customFieldsModulesDto->id = $customFieldsModules->getId();
```

Comment:
@rwhite27 you need to take into account if they user format=true on the listing of custom fields